src/components/account/Account.js

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+
+import Header from '../Header/Header';
+import './style.css'
+import EditDetails from './EditDetails';
+
+
+function createData(key, value) {
+    return { key, value };
+}
+
+const Account = ({ user }) => {
+    const infoKeys = Object.keys(user)
+    const infoValues = Object.values(user)
+    const rows = [
+        createData(infoKeys[0], infoValues[0]),
```

Comment:
Is there a reason for using a lot of magic numbers here? What are you trying to achieve?